Automattic/woocommerce-payments
# International Expansion: Add new countries to supported countries list

Fixes #5535 

#### Changes proposed in this Pull Request

This PR adds new countries to the supported countries list, in addition to the docs links.

**Please do not merge this PR yet**

#### Testing instructions





- Remove your connected WCPay account and clean cache from the Dev plugin
- Switch your branch to develop
- Run `npm run dev`
- Switch your store country to one of those added in this PR, example: Norway
- Try to onboard WCPay. You should get an error like this:

<img width="573" alt="Screenshot 2023-02-22 at 15 53 14" src="https://user-images.githubusercontent.com/12542046/220659918-c4a340a0-4cca-4d1a-a70a-5926677d0492.png">

- Now, get this branch, and try to onboard again. It should let you onboard this time...

Extra steps
- Change your store to a completely different country (Japan for example)
- Try to onboard
- You should get an error like the one ^, but this time the countries added in this PR are also listed

-------------------


**Post merge**



- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : [Link to testing instructions](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions-for-WC-Payments-5.6.0#add-supports-for-12-new-countries)
